.env

```diff
@@ -1 +1,4 @@
-VITE_LIFF_ID=1656508316-k7jNojXm
\ No newline at end of file
+VITE_LIFF_ID=1656508316-k7jNojXm
+
+# TODO
+VITE_LIFF_ID_MINI=
```

Comment:
This lines will be updated by me in a subsequent PR, so it is fine to leave it as a TODO.